data_cleaning/data_cleaning.md

```diff
@@ -0,0 +1,20 @@
+# ROSbot Data Cleaning
+- Once you're done collecting the data, you will need to clean it to remove unwanted parts. 
+- For example, the robot is not moving, or some random guy walks in front of it. 
```

Comment:
do we want to clean all examples of data where a person is in the hallway? that could happen during deployment, right?